frontend/next.config.js

```diff
@@ -5,7 +5,9 @@ const { createSecureHeaders } = require("next-secure-headers");
 
 const isProdBuild = process.env.NODE_ENV === nodeEnv.PRODUCTION;
 
-const SCRIPT_SRC = ["'self'"];
+const PLAUSIBLE_URL = "https://plausible.io";
+
+const SCRIPT_SRC = ["'self'", PLAUSIBLE_URL];
```

Comment:
why do we need to pass `PLAUSIBLE_URL` in the `SCRIPT_SRC`?

Reply:
Great question!

script source to tell browser that it's okay to download and execute the script from Plausible
connect source to tell browser it's okay to connect to Plausible!